Library/Formula/storm.rb

```diff
@@ -1,7 +1,7 @@
 class Storm < Formula
   homepage "http://storm.apache.org"
-  url "http://www.apache.org/dyn/closer.cgi?path=storm/apache-storm-0.9.3/apache-storm-0.9.3.tar.gz"
-  sha1 "33545afc72281d6e8b497fcbdf2b9944eebb83aa"
+  url "https://www.apache.org/dyn/closer.cgi?path=storm/apache-storm-0.9.4/apache-storm-0.9.4.tar.gz"
```

Comment:
Wants to be https: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/23519/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_storm/